packages/font-glyphs/src/letter/armenian/lower-u-group.ptl

```diff
@@ -0,0 +1,347 @@
+$$include '../../meta/macros.ptl'
+
+import [mix linreg clamp fallback] from "@iosevka/util"
+import [DependentSelector] from "@iosevka/glyph/relation"
+
+glyph-module
+
+glyph-block Letter-Armenian-Lower-U-Group : begin
+	glyph-block-import CommonShapes
+	glyph-block-import Common-Derivatives
+	glyph-block-import Letter-Shared-Shapes : nShoulder nShoulderKnots 
+	glyph-block-import Letter-Shared-Shapes : SerifFrame
+	glyph-block-import Letter-Armenian-Shared-Shapes : ArmHBar TwoNeck
+	glyph-block-import Letter-Latin-U : USerifs
+
+	# Common Params
+	define barPos : XH / 2
+	define highBarPos XH
+	define jut Jut
+
+	do "Ben"
+		create-glyph 'armn/ben' 0x562 : glyph-proc
+			local df : include : DivFrame 1
+			include : df.markSet.p
+			include : VBar.l df.leftSB Descender XH df.mvs
+			include : nShoulder
+				left -- (df.leftSB + [HSwToV df.mvs])
+				right -- df.rightSB
+				top -- XH
+				bottom -- (XH / 2)
+				stroke -- df.mvs
+			include : [ArmHBar.normal df 0].base
+			if SLAB : begin
+				local sf : SerifFrame.fromDf df XH Descender
+				include sf.lt.outer
+				if [not para.isItalic] : begin
+					local sf2 : SerifFrame.fromDf df XH (XH / 2)
+					include : composite-proc sf.lb.full sf2.rb.full
+
+	do "Da"
+		create-glyph 'armn/da' 0x564 : glyph-proc
+			local df : include : DivFrame 1
+			include : df.markSet.p
+			include : refer-glyph 'latn/eta'
```

Comment:
This should be moved to `latin/lower-n.ptl` to have more consistent variant behavior. BTW `ss17` uses a tailed `n` under italics which might not be what you want here.

Reply:
Well, if the wiki images for Armenian handwriting is any indication, it's probably fine to have the tail in italic (not that I'm trying to replicate that).

Lower ra is already differentiated by having the n shoulder shift inward.